lib/sequencescape-api/version.rb

```diff
@@ -1,5 +1,5 @@
 module Sequencescape
   class Api
-    VERSION = '0.4.3'.freeze
+    VERSION = '0.5.0-rc1'.freeze
```

Comment:
Is the plan to release another version (without the 'rc1') once it's been through UAT?

Reply:
Yep, or possibly following on from this merge.